### ISSUE TITLE
Tests are failing on version 1.1.0

### ISSUE THREAD

**Issue by MantasVaitkunas** [PAYONE-GmbH/oxid-6#91]
Hi,
with the new Payone release 1.1.0 these errors occurred in the OXID eShop (6.1.x) tests using PayOne module. One of the error:
`Error : Call to a member function isPayOnePaymentType() on null`

To reproduce, you need to run shop tests with active module:
`PARTIAL_MODULE_PATHS=fc/fcpayone ACTIVATE_ALL_MODULES=true RUN_TESTS_FOR_SHOP=true vendor/bin/runtests --filter testSetNewAmountArticleStockControlDerceasingOrderAmountToZero AllTestsUnit`

**Comment by hreinberger** [comment]
Hi @MantasVaitkunas 
thanks for raising this issue. We'll have a look and fix this asap.
best regards

**Comment by hreinberger** [comment]
Hi @MantasVaitkunas 
we just merged #94 which should fix this. Can you confirm?

**Comment by MantasVaitkunas** [comment]
Hi @hreinberger,
your changes fixed some issues, but not all of them, I investigated bit more into shop tests and understood, that some of the changes must be done in the OXID eShop tests, so I have introduced them. From now on looks like integration and unit tests are running successfully.
Unfortunately I saw some failures in selenium tests of OXID eShop. I will investigate those and if changes will have to be done in PayOne module, will create new issue.
